lua/nvim-window.lua

```diff
@@ -223,10 +223,27 @@ end
 
 -- Picks a window to jump to, and makes it the active window.
 function M.pick()
+    local to_remove = {}
   local windows = vim.tbl_filter(function(id)
-    return api.nvim_win_get_config(id).relative == ''
+    local conf = api.nvim_win_get_config(id)
+    if conf.relative == '' then
+      return true
+    end
+    if conf.relative == 'win' and conf.focusable then
+      table.insert(to_remove, conf.win)
+      return true
+    end
+    return false
   end, api.nvim_tabpage_list_wins(0))
 
+  for _, id in pairs(to_remove) do
+    for idx, win_id in pairs(windows) do
+      if win_id == id then
+        table.remove(windows, idx)
+      end
+    end
+  end
+
```

Comment:
A few notes:

1. Please run Stylua on the code because it appears the indentation is messed up
2. Using `vim.tbl_filter` with side effects (= populating an extra map) is a bit ugly, so I'd prefer a regular `for` loop instead
3. Instead of a nested loop to remove the windows, you can do `to_remove[conf.win] = true`, then in the last loop you just check if `to_remove[win_id]` is `true` or not

Reply:
Addressed!

Reply:
I have made the changes and tested it.